library/bignum_mod_raw.h

```diff
@@ -144,6 +144,26 @@ int mbedtls_mpi_mod_raw_write( const mbedtls_mpi_uint *A,
 
 /* BEGIN MERGE SLOT 2 */
 
+/** \brief  Perform a subtraction of two MPIs and return the modulus
+ *          of the result.
+ *
+ * The size of the operation is determined by \p N.
+ *
+ * \param[out] X        The address of the result MPI.
+ *                      This must be initialized. Must have enough limbs to
+ *                      store the full value of the result.
+ * \param[in]  A        The address of the first MPI. This must be initialized.
+ * \param[in]  B        The address of the second MPI. This must be initialized.
```

Comment:
Can `X` be aliased to `B` or not? If not, should be documented. If yes, there should be a test.